logs/log.go

```diff
@@ -261,7 +264,7 @@ func (bl *BeeLogger) Critical(format string, v ...interface{}) {
 		return
 	}
 	msg := fmt.Sprintf("[C] "+format, v...)
-	bl.writeMsg(LevelCritical, msg)
+	bl.writeMsg(LevelCritical, msg, time.Now())
```

Comment:
you should not pass a time.Tim arg into func writeMsg,but new a time.Time object in the writeMsg func